spec/lib/convenient_service/service/plugins/has_result_status_check_short_syntax/concern_spec.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+require "convenient_service"
+
+RSpec.describe ConvenientService::Service::Plugins::HasResultStatusCheckShortSyntax::Concern do
+  example_group "modules" do
+    include ConvenientService::RSpec::Matchers::IncludeModule
+    include ConvenientService::RSpec::Matchers::DelegateTo
+
+    subject { described_class }
+
+    it { is_expected.to include_module(ConvenientService::Support::Concern) }
+
+    context "when included" do
+      subject { service_class }
+
+      before do
+        allow(service_class).to receive(:result).and_return(result)
+      end
+
+      let(:result) { service_class.result }
+
+      let(:service_class) do
+        Class.new.tap do |klass|
+          klass.class_exec(described_class) do |mod|
+            include ConvenientService::Service::Plugins::HasResult::Concern
+            include mod
+
+            def result
+              success
+            end
+          end
+        end
+      end
+
+      specify {
+        expect { service_class.success? }
+          .to delegate_to(result, :success?)
+          .and_return_its_value
+      }
+
+      specify {
+        expect { service_class.error? }
+          .to delegate_to(result, :error?)
+          .and_return_its_value
+      }
+
+      specify {
+        expect { service_class.failure? }
+          .to delegate_to(result, :failure?)
+          .and_return_its_value
+      }
```

Comment:
@Olha-30 

Could you create a dedicated spec file for the `class_methods`, please?

This approach is already used for other plugins. This way can keep plugin specs locations more predictable. 

You find an example in [has_result_short_syntax/concern/class_methods_spec.rb](https://github.com/marian13/convenient_service/blob/main/spec/lib/convenient_service/service/plugins/has_result_short_syntax/concern/class_methods_spec.rb).

Thanks.

Reply:
Oh, actually there is a second approach to do the same thing.

Please, check [has_result_steps//concern_spec.rb](https://github.com/marian13/convenient_service/tree/main/spec/lib/convenient_service/service/plugins/has_result_steps) as well.